File: Examples/android/CMeSdkJava/app_rework/src/main/java/me/connect/sdk/java/sample/db/entity/CredentialOffer.java
package me.connect.sdk.java.sample.db.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import java.io.Serializable;

@Entity
public class CredentialOffer implements Serializable {
    @PrimaryKey(autoGenerate = true)
    public int id;

    @ColumnInfo(name = "connection_id")
    public int connectionId;


    @ColumnInfo(name = "serialized")
    public String serialized;


    @ColumnInfo(name = "claim_id")
    public Integer claimId;

    @ColumnInfo(name = "name")
    public String name;

    // todo temporary
    @ColumnInfo(name = "attributes")
    public String attributes;

    @ColumnInfo(name = "accepted")
    public Boolean accepted;
}

File: Examples/android/CMeSdkJava/app_rework_kt/src/main/kotlin/me/connect/sdk/java/samplekt/db/entity/CredentialOffer.kt
package me.connect.sdk.java.samplekt.db.entity

import androidx.room.ColumnInfo
import androidx.room.Entity

import androidx.room.PrimaryKey


@Entity
data class CredentialOffer(
        @PrimaryKey(autoGenerate = true)
        var id: Int = 0,

        @ColumnInfo(name = "connection_id")
        var connectionId: Int,

        @ColumnInfo(name = "serialized")
        var serialized: String,

        @ColumnInfo(name = "claim_id")
        var claimId: String,

        @ColumnInfo(name = "name")
        var name: String,

        // todo temporary
        @ColumnInfo(name = "attributes")
        var attributes: String,

        @ColumnInfo(name = "accepted")
        var accepted: Boolean? = null,

        @ColumnInfo(name = "message_id")
        var messageId: String
)